crates/sui-types/src/object.rs

```diff
@@ -439,6 +439,9 @@ pub enum Owner {
 }
 
 impl Owner {
+    // Max element size + 8 bytes for enum discriminant
+    pub const SIZE_FOR_GAS_METERING: usize = SuiAddress::LENGTH + 8;
```

Comment:
Is the enum discriminant platform dependent?

Reply:
Couldnt find any official doc on this but my own testing and this [URL](https://www.eventhelix.com/rust/rust-to-assembly-enum-match/#:~:text=The%20size%20of%20the%20enum%20is%2024%20bytes.,currently%20saved%20in%20the%20enum.&text=Note%3A%20A%2064%2Dbit%20discriminator,not%20have%20saved%20any%20memory.) showed its 8 bytes.
Either way when it comes to gas metering sizes, we care more about determinism than precision.

Reply:
I think the relevant encoding for gas metering is BCS, which has 4 byte discriminants

Reply:
>  In BCS, each variant is mapped to a variant index, a ULEB128-encoded 32-bit unsigned integer

(This will be 5 bytes, at a maximum), but in any case -- I don't think there's a huge difference here between 8, 5, or 4, except for confusing future readers.